@here/harp-mapview/lib/poi/PoiManager.ts

```diff
@@ -136,75 +136,21 @@ export class PoiManager {
             );
 
             if (isLineMarkerTechnique(technique) && positions.count > 0) {
-                this.addLineMarker(tile, poiGeometry, technique, techniqueIndex, positions);
+                this.addLineMarker(
+                    tile,
+                    poiGeometry,
+                    technique as LineMarkerTechnique,
+                    techniqueIndex,
+                    positions
+                );
             } else {
```

Comment:
`else if (isPoiTechnique(...))` ?

Reply:
We may add this, but it is impossible to have something else anyway, see condition above:
`
if (!isLineMarkerTechnique(technique) && !isPoiTechnique(technique)) {
   continue;
}
`